zeebe/protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/record/value/user/UserRecord.java

```diff
@@ -27,15 +27,17 @@ public final class UserRecord extends UnifiedRecordValue implements UserRecordVa
   private final StringProperty passwordProp = new StringProperty("password", "");
   private final EnumProperty<UserType> userTypeProp =
       new EnumProperty<>("userType", UserType.class, UserType.REGULAR);
+  private final LongProperty roleKeyProp = new LongProperty("roleKey", -1L);
```

Comment:
❌ A user can have multiple roles assigned, so we should make this an `ArrayProperty`